### PR TITLE
Taxonomy standards subjects

### PR DESCRIPTION
**Pull request by sasZilla**
use authorized user for fetching resources

### CODE REVIEW

**Comment by lucascampbell** [comment]
@sasZilla can you add taxonmy to the search tab so that it is used to return resources.  Please do after lacides fixes his PR, https://github.com/openedinc/opened-api-consumer/pull/3

**Comment by sasZilla** [comment]
@lucascampbell well done

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+url
+exit
+RestClient.get(url.to_s, content)
+url = 'https://partner.opened.com/1/resources?descriptive=&enable_lrmi=false&grade_group=47&limit=12&offset=0&standard_group=2'
+url
+curl 
+RestClient.get(url.to_s, content)
+url = 'https://partner.opened.com/1/resources?category=&descriptive=&enable_lrmi=false&grade_group=47&limit=12&offset=0&standard_group=2'
+RestClient.get(url.to_s, content)
+url
```
Comment (lucascampbell) [inline .byebug_history]: @sasZilla  what is this file? do we need it checked into github?

Comment (sasZilla) [inline .byebug_history]: it's debug log
for sure we don't need this, thanks

**Pull request merged**